roles/avans/tasks/main.yml

```diff
@@ -177,3 +178,43 @@
     - sda
     - sdb
   when: "'galaxy.bioinformatics-atgm.nl' in inventory_hostname"
+
+################
```

Comment:
if you want more organisation here, you could move these to another file in the tasks folder (e.g. `backup.yml`) and and then include that role. https://github.com/galaxyproject/ansible-galaxy/blob/main/tasks/main.yml#L24 ansible-galaxy uses that pattern heavily.